api/MediaKeyStatusMap.json

```diff
@@ -396,10 +396,10 @@
               "version_added": "≤18"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "45"
```

Comment:
`@@iterator` should also be from https://bugzilla.mozilla.org/show_bug.cgi?id=1223980, good.